src/Servers/IIS/IIS/test/Common.FunctionalTests/ShutdownTests.cs

```diff
@@ -422,13 +422,6 @@ public async Task ConfigurationTouchedStress_InProcess()
             await ConfigurationTouchedStress(HostingModel.InProcess);
         }
 
-        [ConditionalFact]
-        [RequiresNewShim]
-        public async Task ConfigurationTouchedStress_OutOfProcess()
```

Comment:
cc @jkotalik

Reply:
Yeah for out of proc this test wasn't really useful and just prone to flakiness. There are scenarios with out of proc that can still cause the app to return 502s during shutdown with out of proc. With inproc, it should be a guarantee.